packages/banner/src/react/__specs__/storyshots.spec.tsx

```diff
@@ -0,0 +1,7 @@
+import path from 'path'
```

Comment:
noticed this was a `tsx` file. i thought we were doing `ts` extensions for this file but i'm not really sure 😄

Reply:
Yeah, in this case, looks like it doesn't need to be tsx.  Some have mocks in them with jsx, those do. Anyway, doesn't apply here.

On the other stories, in the past I:

1. Had an error on createMockNode line
2. Change file extension, and it worked
3. Didn't do strict mode

Now that we're doing strict mode:

1. Had error on createMockNode line
2. Removed that line and the strict tsc error goes away
3. Don't have to redo file ext.

🤷 😆